unified-installer/astra-unified-installer.sh

```diff
@@ -2054,6 +2198,7 @@ step_collect_existing_trident_info() {
 step_existing_trident_flags_compatibility_check() {
     [ "$COMPONENTS" == "$__COMPONENTS_ALL_ASTRA_CONTROL" ] && return 0
     ! existing_trident_needs_modifications && return 0
+    existing_trident_can_be_modified && return 0
```

Comment:
This check was missing from the function, making it think the DO_NOT_MODIFY_EXISTING_TRIDENT flag was always set to true.